polars/polars-core/src/frame/mod.rs

```diff
@@ -2279,6 +2282,106 @@ impl DataFrame {
         Ok(DataFrame::new_no_checks(col))
     }
 
+    /// Summary statistics for a DataFrame. Only summarizes numeric datatypes at the moment and returns nulls for non numeric datatypes.
+    /// Try in keep output similar to pandas
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// # use polars_core::prelude::*;
+    /// let df1: DataFrame = df!("categorical" => &["d","e","f"],
+    ///                          "numeric" => &[1, 2, 3],
+    ///                          "object" => &["a", "b", "c"])?;
+    /// assert_eq!(df1.shape(), (3, 3));
+    ///
+    /// let df2: DataFrame = df1.describe(None);
+    /// assert_eq!(df2.shape(), (8, 4));
+    /// println!("{}", df2);
+    /// # Ok::<(), PolarsError>(())
+    /// ```
+    ///
+    /// Output:
+    ///
+    /// ```text
+    /// shape: (8, 4)
+    /// ┌──────────┬─────────────┬─────────┬────────┐
+    /// │ describe ┆ categorical ┆ numeric ┆ object │
+    /// │ ---      ┆ ---         ┆ ---     ┆ ---    │
+    /// │ str      ┆ str         ┆ f64     ┆ str    │
+    /// ╞══════════╪═════════════╪═════════╪════════╡
+    /// │ count    ┆ null        ┆ 3.0     ┆ null   │
```

Comment:
I think they all should have a count.

Reply:
This was always returning a count sorry I forgot to add the correct output to the doc string. Fixed that now.